doc/source/whatsnew/v0.17.0.rst

```diff
@@ -748,14 +747,14 @@ Previous Behavior:
 
 .. code-block:: ipython
 
-   In [5]: s==None
+   In [5]: s == None  # noqa: E711
```

Comment:
we could ignore this E711 in the docs (but leave for code & tests itself), is this possible to discriminate like this? @datapythonista

Reply:
yes, in `setup.cfg` we have different configurations for `flake8` (code) and `flake8-rst` (docs). SOunds good to me.

@aqurilla can you ignore `E711` for `flake8-rst` there, and remove the `noqa` here please?

Reply:
@datapythonista sure, please have a look,

Reply:
looks good